vmdb/app/controllers/application_controller.rb

```diff
@@ -2451,8 +2451,11 @@ def log_data_size(el, value, indent)
     line = "#{indentation}#{el} <#{value.class.name}> Size #{val_size}"
     line << " Elements #{value.size}"  if value.kind_of?(Hash) || value.kind_of?(Array)
     line << " ActiveRecord Object!!" if value.kind_of?(ActiveRecord::Base)
-    line << " Value #{value.inspect[0...2000]}" unless value.kind_of?(Hash) || value.kind_of?(Array) || value.kind_of?(ActiveRecord::Base)
     $log.warn("MIQ(#{controller_name}_controller-#{action_name}): " + line)
+
+    unless value.kind_of?(Hash) || value.kind_of?(Array) || value.kind_of?(ActiveRecord::Base)
+      $log.debug("Value #{value.inspect[0...2000]}")
```

Comment:
The `if $log.debug?` check is still useful at the end of this logging line to avoid calling inspect on value.  In the issue we were debugging inspect was taking about 5 seconds to complete.  Without that check we are still doing all the that work just not logging the results.